python_modules/dagster/dagster/_core/definitions/run_config.py

```diff
@@ -617,7 +617,7 @@ def _convert_config_classes(configs: Dict[str, Any]) -> Dict[str, Any]:
     return {
         k: {
             "config": config_dictionary_from_values(
-                v._as_config_dict(), v.to_config_schema().as_field()  # noqa: SLF001
+                v._as_config_dict_shallow(), v.to_config_schema().as_field()  # noqa: SLF001
             )
```

Comment:
So if this call site is calling `config_dictionary_from_values` as well, could this just be a "deep" copy?